### DISCUSSION TITLE
Interactive pipeline helper

### DISCUSSION THREAD

**Discussion by pierrethibault** [ptycho/ptypy#444] [Ideas]
When working interactively it would be convenient to provide a helper function that guides the user to the next steps in the Ptycho preparation.

**Comment by alexbjorling** [comment]
People here at Max IV are very interested in "pipeline helpers" that would allow users to easily build and modify their analysis workflows. They seem to think Orange 3 is a good way to go, based on the example of a ray tracing program from the ESRF (http://ftp.esrf.eu/pub/scisoft/Oasys/readme.html).

I know nothing about Orange but seems it might be an option.
